packages/duktape/build.sh

```diff
@@ -0,0 +1,36 @@
+TERMUX_PKG_HOMEPAGE=https://www.duktape.org/
+TERMUX_PKG_DESCRIPTION="The Duktape JavaScript interpreter"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_VERSION=2.3.0
+TERMUX_PKG_SKIP_SRC_EXTRACT=1
+TERMUX_PKG_BUILD_IN_SRC=yes
+
+termux_step_extract_package() {
+	git clone --depth=1 https://github.com/svaarala/duktape.git -b v${TERMUX_PKG_VERSION} ${TERMUX_PKG_SRCDIR}
+}
+
+termux_step_post_extract_package() {
+	sudo apt-get -yq update
+	sudo apt-get install -yq python2.7 python-pip
+	pip install pyyaml
```

Comment:
Privileged (sudo) package installation is not allowed for build scripts.

Reply:
Packages `python2.7` and `python-pip` should be installed in Docker image.
See script [scripts/setup-ubuntu.sh](https://github.com/termux/termux-packages/blob/master/scripts/setup-ubuntu.sh).

Reply:
Done!